### DISCUSSION TITLE
i got this!

### DISCUSSION THREAD

**Discussion by Zilaeed03** [Vedza/NitroSniperGo#256] [General]
Hello 
so in webhook channel i got this 
Nitro Sniped !
Code
uXRfUxB6WWjgAYM9
Guild
Evo
Channel
┃general-chat
Response
This gift has been redeemed already. does this mean that i have 2 nitro or not?

**Comment by Asvarius** [comment]
hello @Zilaeed03, you didn't got any codes actually, it's saying that the code catched is not valid. I recommend you to create multiple accounts as alts and enter in 100 server in each. :)

**Reply by Zilaeed03** [reply]
Hey there is any way I could write to u on discord @Asvarius or no? Bc i got some questions to ask u

**Reply by Asvarius** [reply]
yeah sure, what's your discord?
